Source/DafnyLanguageServer.Test/Various/ConcurrentInteractionsTest.cs

```diff
@@ -0,0 +1,181 @@
+﻿using Microsoft.Dafny.LanguageServer.IntegrationTest.Extensions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Various {
+  [TestClass]
+  public class ConcurrentInteractionsTest : DafnyLanguageServerTestBase {
+    // Implementation note: These tests assume that no diagnostics are published
+    // when a document (re-load) was canceled (DafnyDocument.LoadCanceled).
```

Comment:
cancelled

Reply:
That's British vs American English. :) Since most terms within Dafny are American English and .NET's terminology uses *canceled* too (see `OperationCanceledException`), I think we should stick to this. Or did I misinterpret this comment?

Reply:
Ah, you're right. My bad :)